web/src/main/kotlin/no/nav/su/se/bakover/web/Extensions.kt

```diff
@@ -21,7 +21,7 @@ internal fun ApplicationCall.audit(msg: String) {
     sikkerLogg.info("${lesBehandlerId()} $msg")
 }
 
-internal fun ApplicationCall.lesBehandlerId() =
+private fun ApplicationCall.lesBehandlerId() =
     (this.authentication.principal as JWTPrincipal).payload.getClaim("oid").asString()
```

Comment:
Vad bruks denna till nå när vi använder/kan använda nav-identen?

Reply:
Den brukes til audit-logging. Se kommentar over/under for diskusjon rundt bruk av `navIdent`.